test/functional/test_framework/util.py

```diff
@@ -497,7 +497,15 @@ def check_node_connections(*, node, num_in, num_out):
     assert_equal(info["connections_out"], num_out)
 
 def fill_mempool(test_framework, node, miniwallet):
-        """Fill mempool until eviction."""
+        """Fill mempool until eviction.
+
+        Allows for simpler testing of scenarios with minfee != minrelay
+        Requires -datacarriersize=100000 and
+        -maxmempool=5(or similarly small).
```

Comment:
"or similarly small" doesn't really make sense to me. AFAICT this makes just enough transactions to go past the 5MB mempool.

Reply:
removed the parenthesis